### ISSUE TITLE
Feedback API not called when using Models

### ISSUE THREAD

**Issue by smolina74** [SeldonIO/seldon-core#251]
This is the configuration for the runtime graph I have:
`"graph": {
                   "children": [
                   ],
                   "endpoint": {
                       "service_host": "0.0.0.0",
                       "service_port": 9000,
                       "type": "REST"
                   },
                   "methods": [
                       "TRANSFORM_INPUT",
                       "SEND_FEEDBACK"
                   ],
                   "name": "digitclfeedback-1-1-0",
                   "type": "MODEL"
               }
`
This is my model (predictor) implementation:

from sklearn.externals import joblib
import numpy as np
`
 class Model:
    def __init__(self):
        self.clf = joblib.load('/data/digitsclassifier-1-1-0/SVM.pkl') 

    def predict(self,X,feature_names):
        # X is a numpy array containing dictionaries of the form
        # {'id':<ID>, 'values': <features>}
        # So each dictionary contain the sample to be use by one prediction plus
        # some identification identifying that sample.
        ids = [elem['id'] for elem in X]
        feature_matrix = np.array([elem['values'] for elem in X])
        predictions = self.clf.predict(feature_matrix)
        predictions_with_ids = [{'id': elem[0], 'values': elem[1]} for elem in zip(ids, predictions)]
        print(predictions_with_ids)
        return predictions_with_ids

    def send_feedback(self, Xsamples, feature_names, Xtruth, reward):
        # Xsamples is a numpy array containing dictionaries of the form
        # {'id':<ID>, 'values': <features>}, which represents the original samples
        # for which a prediction was requested.
        # Xtruth is also a numpy array containing dictionaries of the form 
        # {'id':<ID>, 'values': <prediction>}, which represents the predictions
        # values according to the predictor's user (feedback)
        # reward is just a numeric value indicating a reward in the Reinforcement Learning system.        
        print('Original Request {}'.format(Xsamples))
        print('Truth {}'.format(Xtruth))
        print('Reward {}'.format(reward))
`

When I send a feedback request using the EXTERNAL API I can see that the message arrives to the Seldon Container Engine but  it is not forwarded to the companion model container. This is my request:
`curl --header "Content-Type: application/json" --request POST --data @json_request.json http://localhost:8002/seldon/digitclfeedback-1-1-0/api/v0.1/feedback`

Also I can see this message in the Seldon Container Engine log:
**NODE digitclfeedback-1-1-0: entered feedback**

(by the way I can use the predict API without issues)

**Comment by cliveseldon** [comment]
Fixed by #254 
You will need to rewrap your models with 0.2.4-SNAPSHOT wrappers and use 0.2.4-SNAPSHOT images.
There is an example [here](https://github.com/SeldonIO/seldon-core/blob/master/examples/models/template_model_with_feedback/modelWithFeedback.ipynb)

**Comment by vackysh** [comment]
Hi @cliveseldon  @smolina74 ,

Sorry to open this thread again, i have the similar issue aroused and unable to get anything from the send_feedback API.  I am sending payload request through postman API

{
    "request": {
		"data": {
			"ndarray": [
				[
					-0.360401125437509,
					-1.6118801888587946,
					4.578304752033222,
					-0.6537760204057684,
					-0.8343145759984982,
					1.117818816508468,
					0.5250776596051238,
					-0.022866028481639277,
					-0.997698231112318,
					1.3715932476110648,
					-0.34868264699412893,
					-0.49288047614152425
				],
				[
					2.7762444096949888,
					0.6203935049961723,
					-0.2641776865169515,
					-0.26441276083322596,
					-0.8343145759984982,
					0.0696409382326457,
					-0.3541843043004386,
					-0.6884393702300454,
					1.1832526792948979,
					-1.2232972516480227,
					-0.34868264699412893,
					0.49361481432592424
				],
				[
					2.7762444096949888,
					0.6203935049961723,
					-0.24608622037043779,
					1.0307550242583157,
					-0.8343145759984982,
					-0.9785369400431766,
					1.4043396235106862,
					0.275033793803416,
					1.211761841522443,
					1.9704141320554696,
					-0.34868264699412893,
					-0.6960790094245382
				]
			]
		}
	},
	"response": {
        "data": {"ndarray": [["Paid Off"]]}
        },
    "reward": 1.0
} 

here are predict and send_feedback methods defined

    def send_feedback(self,X, feature_names, reward, truth, routing=None):
        print("Send feedback called")
        print('Original Request {}'.format(X))
        print('features names {}'.format(feature_names))
        print('Truth {}'.format(truth))
        print('Reward {}'.format(reward))

Please let me know if my payload structure is correct. If not, please assist me in creating correct payload. 

Regards,
Varun
